lib/orogen/templates/proxies/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Generated from orogen/lib/orogen/templates/proxies/CMakeLists.txt
+
+
+SET(PROXYLIB_NAME <%= component.name %>-proxies)
+
+add_definitions(-DRTT_COMPONENT)
+
+<% 
+    pkg_conf_deps = ["orocos_cpp_base"]
+    pkg_conf_deps.concat(dependencies(false))
```

Comment:
Avoid calling methods without receivers in the template code, it gets very confusing very quickly. That's why there is a 'project' local variable defined for the main templates (even though they could use 'self').